README.md

```diff
@@ -85,3 +85,9 @@ For easier use, you can use the `StencilContext.enrich(context:parameters:enviro
 
 - `param`: the parsed parameters using the parser mentioned above.
 - `env`: a dictionary with all available environment variables (such as `PATH`).
+
+---
+
+# Licence
```

Comment:
License/licence we should decide once and for all of we're US or UK 😛

Reply:
Most of the existing spelling in the project is the British one, so let's try to keep it.